index.html

```diff
@@ -393,7 +393,7 @@ <h1 class="contact_form_heading">
         </p>
       </div>
 
-      <form action="https://formspree.io/f/mpzkoegn" method="post">
```

Comment:
[  OPTIONAL ] Please add a project screenshot to README so that users can view the project without actually cloning it.